app/src/main/java/com/example/githubuserbrowser/core/network/retrofit/RetrofitUserDataSource.kt

```diff
@@ -16,7 +16,7 @@ import javax.inject.Singleton
 
 private interface RetrofitApi {
     @GET("users")
-    suspend fun getUsers(@Query("page") page: Int, @Query("per_page") perPage: Int): List<UserDto>
+    suspend fun getUsers(@Query("since") since: String, @Query("per_page") perPage: Int): List<UserDto>
```

Comment:
The since query is an Integer. https://docs.github.com/en/rest/users/users?apiVersion=2022-11-28#list-users